### PR TITLE
feat: add maps.SortedKeys

### PR DESCRIPTION
**Pull request by caarlos0**
added maps.SortedKeys, can add more methods in the future.

This one simply combines `x/maps.Keys` and `x/slices.Sort`, and is merely a convenience.

### CODE REVIEW

File: exp/maps/maps.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package maps
+
+import (
+	"golang.org/x/exp/constraints"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
```
Comment (aymanbagabas) [inline exp/maps/maps.go:6]:
```suggestion
	"slices"
```

File: exp/maps/maps.go [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+)
+
+// SortedKeys returns the keys of the map m.
+// The keys will be sorted.
+func SortedKeys[M ~map[K]V, K constraints.Ordered, V any](m M) []K {
```
Comment (aymanbagabas) [inline exp/maps/maps.go:11]:
```suggestion
func SortedKeys[M ~map[K]V, K cmp.Ordered, V any](m M) []K {
```

File: exp/maps/maps.go [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package maps
+
+import (
+	"golang.org/x/exp/constraints"
```
Comment (aymanbagabas) [inline exp/maps/maps.go:4]:
```suggestion
	"cmp"
```

File: exp/maps/maps_test.go [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package maps
+
+import (
+	"testing"
+
+	"golang.org/x/exp/slices"
```
Comment (aymanbagabas) [inline exp/maps/maps_test.go:6]:
```suggestion
	"slices"
```

**Review by aymanbagabas — COMMENTED** [review]
We could use standard library `slices` and `cmp` packages to limit the use of `x/exp`

File: exp/maps/maps.go (hunk 1 above)
Comment (caarlos0) [reply exp/maps/maps.go:6]: will require a highger go version... which is fine I think

**Review by ccoVeille — APPROVED** [review]

**Pull request merged**